src/scrape_up/github/respository.py

```diff
@@ -351,6 +354,19 @@ def get_contributors(self):
                 "message": message,
             }
 
+
+    def last_update_at(self):
+        data=self.__scrape_page()
+        try:
+            update=data.find_all("relative-time", class_="no-wrap")
+            return update[0].get_text()
+        except:
+            message="Oops! No Repo or Organization found"
+            return message
+    
```

Comment:
Do follow the return type convention of the project, and check other methods for reference.

Reply:
I have done it from my end. Now, you can check it and merge it.